cmd/corald/main.go

```diff
@@ -48,7 +48,7 @@ func main() {
 		writeTimeout = 30 * time.Second
 	)
 
-	if err := app.Run(":4001", routes.API(), readTimeout, writeTimeout); err != nil {
+	if err := app.Run(":3000", routes.API(), readTimeout, writeTimeout); err != nil {
```

Comment:
It should be using it as Run loads it. See https://github.com/ardanlabs/kit/blob/master/web/app/app.go#L196-L200 if you want to see what's going on under the hood :)